src/Pages/AboutUs/about-us.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { MyHeader } from '../../components/MyHeader/my-header'
```

Comment:
Eslint should be used here to avoid such inconsistencies as sporadic semicolon usage on lines 1 & 2.